pyaedt/hfss.py

```diff
@@ -40,52 +40,51 @@ class Hfss(FieldAnalysis3D, object):
         the active version or latest installed version is used.
     NG : bool, optional
         Whether to run AEDT in the non-graphical mode. The default
-        is ``False``, which launches AEDT in the graphical mode.
+        is ``False``, in which case AEDT is launched in the graphical mode.
     AlwaysNew : bool, optional
         Whether to launch an instance of AEDT in a new thread, even if
         another instance of the ``specified_version`` is active on the
         machine. The default is ``True``.
     release_on_exit : bool, optional
         Whether to release AEDT on exit. The default is ``False``.
-        Whether to release AEDT on exit. The default is ``True``.
     student_version : bool, optional
         Whether to open the AEDT student version. The default is 
         ``False``.
 
     Examples
     --------
 
-    Create an instance of `HFSS` and connect to an existing HFSS
-    design or create a new `HFSS` design if one does not exist.
+    Create an instance of HFSS and connect to an existing HFSS
+    design or create a new HFSS design if one does not exist.
```

Comment:
@PipKat @akaszynski, what should be the convention here going forward? @MaxJPRey recently added these examples and wrapped the HFSS name with tick marks. Just so we are all on the same page moving forward, should object types in example descriptions not be wrapped with tick marks?

Reply:
@jazztekk @akaszynski 
I've been removing them because it looks better -- but I was assuming it was OK to use as generic reference to the Ansys product. It it should be clear that it's an object name, it should be one back tick and same capitalization used for the object, such as 'Hfss`. I'm OK with whatever @akaszynski  says and will redo accordingly.